src/main/java/org/opensearch/neuralsearch/query/NeuralSparseQueryBuilder.java

```diff
@@ -118,10 +132,15 @@ protected void doWriteTo(StreamOutput out) throws IOException {
     protected void doXContent(XContentBuilder xContentBuilder, Params params) throws IOException {
         xContentBuilder.startObject(NAME);
         xContentBuilder.startObject(fieldName);
-        xContentBuilder.field(QUERY_TEXT_FIELD.getPreferredName(), queryText);
+        if (Objects.nonNull(queryText)) {
```

Comment:
"" is acceptable here?

Reply:
yes

Reply:
Although it's an invalid query to use an empty string and we'll throw exception. Here we accept empty string

Reply:
will add check logics in `fromXContent` to make sure the queryText and modelId is nonNull from user